index.html

```diff
@@ -0,0 +1,342 @@
+                    <h4 class="feature__title">CUSTOMIZABLE</h4>
+                    <p class="feature__description">Change the colors, pictures or any of the sections to suit your
+                        needs.</p>
+                </div>
+                <div class="feature">
+                    <div class="feature-icon__container">
+                        <img
+                            class="feature__icon"
+                            src="./images/design.svg"
+                            alt="Design icon"
+                        >
+                    </div>
+                    <h4 class="feature__title">SLICK AND BEAUTIFUL DESIGN</h4>
+                    <p class="feature__description">Trendy and fresh design, fits any website.</p>
+                </div>
+            </div>
+
+        </section>
+
+        <aside class="logos">
+            <div class="logos__container container">
+                <img
+                    class="logos__image"
+                    src="./images/logos.png"
+                    alt="Logos list image"
+                >
+            </div>
+        </aside>
+
+        <section class="web-project">
+            <figure class="web-project__container-image">
+                <img
+                    class="web-project__image"
+                    src="images/imac.png"
+                    alt="iMac image"
+                >
+            </figure>
+            <article class="web-project__text">
+                <header class="web-project__header title">SIMPLE AND BEAUTIFUL</header>
+                <p class="web-project__description description">Use Nova theme for your next web project. It is
+                    completely
+                    customizable so you can change any of the
+                    sections to fit your needs. Nova Theme is Free for any kind of use, personal and commercial. Have
+                    fun and good luck!</p>
+            </article>
+        </section>
+
+        <section
+            class="pricing__box"
+            id="pricing"
+        >
+            <article class="pricing__container container">
+                <div class="pricing__header">
+                    <h2 class="pricing__title title">CHOOSE YOUR PRICING PLAN</h2>
+                    <p class="pricing__description description">Pick any of our super affordable pricing plans</p>
+                </div>
+            </article>
+
+
+            <div class="pricing-plans container">
+
+                <div class="pricing-plan pricing-plan-1">
+                    <p class="pricing-plan__version">BASIC</p>
+                    <h3 class="pricing-plan__price">$15</h3>
+                    <p class="pricing-plan__m">per month</p>
+                    <div class="pricing-plan__feature-container">
+                        <p class="pricing-plan__feature">Up to 7 Projects</p>
+                        <p class="pricing-plan__feature">2 Additional Developers</p>
+                    </div>
```

Comment:
Zacytowanie zrobiłbym to jako `ul > li` bo to lista elementów jaką otrzymujemy w danym pakiecie.